Formula/w/wpebackend-fdo.rb

```diff
@@ -6,6 +6,14 @@ class WpebackendFdo < Formula
   license "BSD-2-Clause"
   head "https://github.com/Igalia/WPEBackend-fdo.git", branch: "master"
 
+  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
+  # labeled as "pre-release" on GitHub before the version is released, so it's
+  # necessary to use the `GithubLatest` strategy.
```

Comment:
```suggestion
```

The `stable` URL is a GitHub release asset, so we don't need to document `GithubLatest` here. That said, if we wanted to explain it anyway, I would use something like this:

```
  # Tags with a 90+ patch are unstable and they are marked as pre-release on
  # GitHub. The `stable` archive is a GitHub release asset, so we use the
  # `GithubLatest` strategy to avoid unstable releases.
```

Reply:
let's remove the comment then